assets/js/components/HostsList.jsx

```diff
@@ -177,6 +182,21 @@ function HostsList() {
           />
         ),
       },
+      {
+        title: '',
+        key: 'deregisterable',
+        className: 'w-48',
+        render: (content, item) =>
+          content ? (
```

Comment:
What do you think about conditional rendering using `content && <CleanUpButton ... />` here?